php/blocks/class-loader.php

```diff
@@ -187,7 +187,7 @@ public function get_block_attributes( $block ) {
 			}
 		}
 
-		return $attributes;
+		return apply_filters( 'block_lab_get_block_attributes', $attributes );
```

Comment:
@lukecarbis, would you mind if I committed a DocBlock for `apply_filters` here, including a `@param` tag for `$attributes`?

Reply:
Of course! I should have thought of that. Go for it.

Reply:
Thanks!